src/elements/content-preview/ContentPreview.js

```diff
@@ -1050,6 +1050,7 @@ class ContentPreview extends PureComponent<Props, State> {
     render() {
         const {
             apiHost,
+            features,
```

Comment:
can you cherry-pick this commit as well: https://github.com/box/box-ui-elements/pull/792/commits/97dd553fdad920d455d8ec7fa002d4fa816e7874

Reply:
@alexkrolick  Would the features need to be update to match the ones coming from EUA?
```
    options.features = {
        activityFeed: {
            avatars: appFeatures.inbox,
            tasks: {
                createButton: appFeatures.inbox,
                createFromComment: !appFeatures.inbox,
            },
        },
    };
```

Reply:
yep